tests/unittest/_torch/distributed/test_ipc_memory_fallback.py

```
@@ -0,0 +1,320 @@
+
+from tensorrt_llm import _ipc_utils
+from tensorrt_llm._ipc_utils import IpcMemory
+from tensorrt_llm.functional import AllReduceStrategy
+from tensorrt_llm.mapping import Mapping
+
+
+def _has_allreduce_op() -> bool:
+    """Whether the trtllm custom ops are registered in this build."""
+    try:
+        return torch.ops.trtllm.allreduce is not None
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)